src/database/repositories/recipe.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+class RecipeRepository {
+  constructor(prisma) {
+    this.prisma = prisma;
+  }
+
+  async add(name, categoryId, products, description, imageLink) {
```

Comment:
You can use sort of DTO object instead of passing all the fields as params

Reply:
changed it to use DTO